extensions/kubeapi/rbac/update.go

```diff
@@ -42,3 +42,71 @@ func UpdateGlobalRole(client *rancher.Client, updatedGlobalRole *v3.GlobalRole)
 	}
 	return newGlobalRole, nil
 }
+
+// UpdateRoleTemplate is a helper function that uses the dynamic client to update an existing cluster role template
+func UpdateRoleTemplate(client *rancher.Client, updatedCRT *v3.RoleTemplate) (*v3.RoleTemplate, error) {
```

Comment:
To keep this function generic that can be used to update role template in both cluster and project contexts, can you rename `updatedCRT` to `updatedRoleTemplate`?